diff -ur a/setup.cfg b/setup.cfg
--- a/setup.cfg	2023-07-05 01:00:16.755437100 -0400
+++ b/setup.cfg	2023-09-25 04:10:54.032293678 -0400
@@ -71,7 +71,6 @@
 	scikit-image[data]>=0.19.1 # just `pooch`, but needed by `builtins` to provide all scikit-image.data samples
 	scipy>=1.4.1 ; python_version < '3.9'
 	scipy>=1.5.4 ; python_version >= '3.9'
-	sphinx>=4.3.0,<5  # numpydoc dependency. sphinx>=5 breaks the docs build; see https://github.com/napari/napari/pull/4915
 	superqt>=0.4.1
 	tifffile>=2020.2.16
 	toolz>=0.10.0
